ZotBinsAPI.py

```diff
@@ -282,7 +289,94 @@ def get_obervation_stats():
         print(e)
         return str(e)
 
+@app.route('/mocked/observation/get', methods=['GET'])
+def mocked_get_observation():
+    """ 
+    An API endpoint that mocks observation GET requests.
+
+    Used for web and mobile applications to simulate
+    an identical GET request for real data.
+
+    Must pass all required params.
+
+    Possible sensor types = ["B", "D", "W", "DI"]
+        "B"  - Breakbeam
+        "D"  - Distance (ultrasonic sensor)
+        "W"  - Weight (load cell)
+        "Di" - Diversion Rate
+
+    @param   sensor_id, format = "ZBin" + [SENSOR_ID] + [SENSOR_TYPE]
+    @param   start_timestamp, format = "YYYY-MM-DD HH:MM:SS"
+    @param   end_timestamp, format example = "2020-02-01 9:30:01"
+    @return  JSON-ified array of mocked datapoints
+    """
+    try:
+        con = pymysql.connect(config.host, config.user, config.pw, config.db, cursorclass=pymysql.cursors.DictCursor)
+        ret = []
+        if request.method == 'GET':
+            sensor_id = request.args.get("sensor_id")
+            start_timestamp = request.args.get("start_timestamp")
+            end_timestamp = request.args.get("end_timestamp")
+
+            """
+            Perform a quick null check and return
+            useful info to the caller so that they
+            know what fields they should be adding.
+            
+            TODO: Probably should have this null check 
+            anytim these params are being used.
+            """
+            if (sensor_id == None):
+                raise Exception("Param 'sensor_id' cannot be null")
+            if (start_timestamp == None):
```

Comment:
personal preference is `elif`. Logic is same though so up to you.

Reply:
Sure. I don't use elif's here because `elif` to me is for a block like this:
`if (sensor_id == None): ...
elif (sensor_id == "Josh"): ...
elif (sensor_id == "Jesse"): ...
else: ...`

Since `sensor_id`, `start_timestamp`, and `end_timestamp` are different var's, I don't think it's appropriate to use elif here.